tests/images/run.sh

```diff
@@ -13,47 +13,45 @@
-./bin/image-classifier tests/images/imagenet/*.png -image-mode=0to255 -m=zfnet512 -model-input-name=gpu_0/data "$@"
-./bin/image-classifier tests/images/imagenet/*.png -image-mode=0to1 -m=densenet121 -model-input-name=data -compute-softmax "$@"
-./bin/image-classifier tests/images/imagenet/*.png -image-mode=0to1 -m=shufflenet -model-input-name=gpu_0/data "$@"
-./bin/image-classifier tests/images/mnist/*.png -image-mode=0to1 -m=lenet_mnist -model-input-name=data "$@"
-./bin/image-classifier tests/images/imagenet/*.png -image-mode=0to255 -m=inception_v1 -model-input-name=data "$@"
-./bin/image-classifier tests/images/imagenet/*.png -image-mode=0to255 -m=bvlc_alexnet -model-input-name=data "$@"
-for png_filename in tests/images/imagenet/*.png; do
-  ./bin/image-classifier "$png_filename" -use-imagenet-normalization -image-mode=0to1 -m=resnet50/model.onnx -model-input-name=gpu_0/data_0 "$@"
-done
-for png_filename in tests/images/imagenet/*.png; do
-  ./bin/image-classifier "$png_filename" -image-mode=neg128to127 -m=vgg19/model.onnx -model-input-name=data_0 "$@"
-done
-for png_filename in tests/images/imagenet/*.png; do
-  ./bin/image-classifier "$png_filename" -image-mode=neg128to127 -m=squeezenet/model.onnx -model-input-name=data_0 "$@"
-done
-for png_filename in tests/images/imagenet/*.png; do
-  ./bin/image-classifier "$png_filename" -image-mode=0to255 -m=zfnet512/model.onnx -model-input-name=gpu_0/data_0 "$@"
-done
-for png_filename in tests/images/imagenet/*.png; do
-  ./bin/image-classifier "$png_filename" -image-mode=0to1 -m=densenet121/model.onnx -model-input-name=data_0 -compute-softmax "$@"
-done
-./bin/image-classifier tests/images/imagenet/zebra_340.png -image-mode=0to1 -m=shufflenet/model.onnx -model-input-name=gpu_0/data_0 "$@"
-for png_filename in tests/images/mnist/*.png; do
-  ./bin/image-classifier "$png_filename" -image-mode=0to1 -m=mnist.onnx -model-input-name=data_0 "$@"
-done
-for png_filename in tests/images/imagenet/*.png; do
-  ./bin/image-classifier "$png_filename" -image-mode=0to255 -m=inception_v1/model.onnx -model-input-name=data_0 "$@"
-done
-for png_filename in tests/images/imagenet/*.png; do
-  ./bin/image-classifier "$png_filename" -image-mode=0to255 -m=bvlc_alexnet/model.onnx -model-input-name=data_0 "$@"
+imagenetFiles=("cat_285" "dog_207" "zebra_340")
+imagenetIdxValue=(281 207 340)
+inceptionIdxValue=(173 124 79)
+
+imagenet299Files=("cat_281_299" "dog_207_299" "zebra_340_299")
+googlenetV1IdxValue=(281 222 340)
+googlenetV4IdxValue=(281 207 340)
+
+mnistFiles=("0_1009" "1_1008" "2_1065" "3_1020" "4_1059" "5_1087" "6_1099" "7_1055" "8_1026" "9_1088")
+mnistIdxValue=(0 1 2 3 4 5 6 7 8 9)
+
+for i in ${!imagenetFiles[@]}; do
+	./bin/image-classifier tests/images/imagenet/${imagenetFiles[$i]}.png -idx=${imagenetIdxValue[$i]} -use-imagenet-normalization -image-mode=0to1 -m=resnet50 -model-input-name=gpu_0/data "$@"
```

Comment:
So the image-classifier returns non-zero in the case of error, can we make this script also return non-zero if any run fails, that way we can use it as part of the CI system.

Reply:
script has instruction:
set -e
it means whatever environment runs script will see script exits on error with the correspondent error code.

Reply:
Yes, but `set -e` early exits. Similar to the unit tests, it would be better to get the full list of failures in one run so they can be fixed in a single commit.

Reply:
Fixed in the last commit